lambda/templates/config/jest.config.base.ts

```diff
@@ -0,0 +1,18 @@
+import { Config } from "jest";
+
+export default {
+  preset: "ts-jest",
+  clearMocks: true,
+  modulePaths: ["<rootDir>/src"],
+  collectCoverageFrom: ["<rootDir>/src/**/*"],
+  testPathIgnorePatterns: ["./build/*"],
+  testMatch: ["<rootDir>/tests/*"],
+  coverageThreshold: {
+    global: {
+      statements: 100,
+      branches: 100,
+      functions: 100,
+      lines: 100,
+    },
+  },
+} satisfies Config;
```

Comment:
Ok this is neat, I wondered want this was so it's to ensure type safety; we do something similar https://github.com/govuk-one-login/ipv-cri-check-hmrc-api/blob/main/integration-tests/jest.config.common.ts#L1-#L3 it is more obvious what's going wonder which is better and whether `satisfies` has any obvious benefits over the method we used. I suppose once you know what `satisfies` does it fine. However our method is more direct approach

Reply:
`satisfies` is standard TypeScript syntax. It tells the typechecker to verify that the struct conforms to the given interface without changing its resulting type. It is the correct use case for this operator and there's a more detailed explanation in the [TypeScript documentation](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html).
In practice, both approaches would work but I think `satisfies` does what we need here - verify that the config is correct and doesn't have any type errors.
And yes, there's full IDE support and intellisense for this since this is part of the TS spec.